### DISCUSSION TITLE
PCSC in Solo Key

### DISCUSSION THREAD

**Discussion by jaclas** [solokeys/solo2#173] [Q&A]
I'm browsing the forum to learn how to use Solo for SSH keys and noticed that everyone gets something like this after typing the _solo2 ls_ command:

`USB-A : Solo 2 88E370EEB9BF535D8BE6******** (CTAP+PCSC, firmware 2:20220822.0, unlocked)`

And in my case it is like this:

`Solo 2 6E204A6C139A945890BC******** (CTAP only, firmware 2:20220822.0, locked)`

Why don't I have +PCSC? What is PCSC and how add this?

I use Ubuntu 22.10

**Comment by magiruuvelvet** [comment]
I had the same issue. After installing [`ccid`](https://github.com/LudovicRousseau/CCID) the `solo2 ls` command suddenly started to show `CTAP+PCSC` instead of just `CTAP only`. I actually was setting up a Nitrokey and had to install a CCID driver. I'm not 100% sure, but I think installing it enabled PCSC support in the `solo2` cli.

You also need to install [`pcsc-lite`](https://github.com/LudovicRousseau/PCSC) if you do not have it already.

I'm on Gentoo though. Refer to the Ubuntu repositories how the CCID and PCSC packages are named.

**Comment by jaclas** [comment]
Unfortunately, installing both tools did not change anything. To be sure, I compiled from the sources to have the latest versions, because there were older versions in the repositories.
